examples/soundcloud.amp.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Soundcloud examples</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async custom-element="amp-soundcloud" src="/dist/v0/amp-soundcloud-0.1.max.js"></script>
```

Comment:
I'll try to make it clear:

In the file in the `examples` dir (this one). All URLs should pretend they point to production. So start with `https://cdn.ampproject.org` and so forth. Thus when you first submit this, this example doesn't yet work. But we want them to be copy pasteable – that is why they point to prod, because that is what the users need.

We then automatically create files for you in `examples.build` directory that point to the local standard build file (That is the .max) and the local build minified file (.min). The latter only works when you run `gulp dist`.

So, don't worry would any file names here, but run `examples.build/soundcloud.amp.max.html` for testing.